core/src/blocktree_processor.rs

```diff
@@ -205,12 +220,14 @@ pub fn process_blocktree(
         }
 
         if !entries.is_empty() {
-            if !entries.verify(&last_entry_hash) {
-                warn!(
-                    "Ledger proof of history failed at slot: {}, entry: {}",
-                    slot, entry_height
-                );
-                return Err(BlocktreeProcessorError::LedgerVerificationFailed);
+            if rollback_height.is_none() {
```

Comment:
Can you please make skipping the verify a separate option to `process_blocktree...`, I'll have use for it soon outside of ledger rollback  :)